server/controllers/userController.js

```diff
@@ -5,7 +5,20 @@ import User from "../models/userModel.js";
 // @route   POST /api/users/auth
 // @access  Public
 const authUser = asyncHandler(async (request, response) => {
-  response.status(200).json({ message: "User authentication successful" });
+  const { email, password } = request.body;
+
+  const user = await User.findOne({ email });
+
+  if (user && (await user.matchPassword(password))) {
+    response.status(201).json({
```

Comment:
Can we change the status code to 200? 201 means that a resource is created.

Reply:
Done! Merging